packages/examples/babylon-typescript-agent/README.md

```
@@ -236,20 +236,20 @@ Live server verification:
 The agent tries providers in order:
 1. **Groq** (primary) - Fast inference with `llama-3.1-8b-instant`
 2. **Claude** (fallback) - `claude-sonnet-4-5`
-3. **OpenAGI** (fallback) - `gpt-5-nano`
+3. **OpenAI** (fallback) - `gpt-5.1`
 
 Provide at least one API key.
 
 ## Development
 
 ### Run Tests
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)